code/obj/machinery/computer.dm

```diff
@@ -12,19 +12,77 @@
 	/// does it have a glow in the dark screen? see computer_screens.dmi
 	var/glow_in_dark_screen = TRUE
 	var/image/screen_image
+
+	var/can_reconnect = 0 //Set to 1 to make multitools call connection_scan. For consoles with associated equipment (cloner, genetek etc)
+	var/obj/item/circuitboard/circuit_type = null
+	Topic(href, href_list)
+		if (..(href, href_list))
+			return 1
+		playsound(src.loc, 'sound/machines/keypress.ogg', 30, 1, -15)
+
+	attack_hand(var/mob/user)
+		..()
+		if (!user.literate)
+			boutput(user, "<span class='alert'>You don't know how to read or write, operating a computer isn't going to work!</span>")
+			return 1
+		interact_particle(user,src)
+
+	attack_ai(mob/user as mob)
+		src.attack_hand(user)
+
+	attackby(obj/item/W as obj, mob/user as mob)
+		if (can_reconnect)
+			if (istype(W, /obj/item/device/multitool) && !(status & (BROKEN|NOPOWER)))
```

Comment:
Don't check for `/obj/item/device/multitool`, I don't think? e.g. cyborg/syndicate omnitool won't work here. Should probably be a `ispulsingtool` check.

Reply:
Done!